.gitignore

```diff
@@ -0,0 +1,61 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
+lib64/
+parts/
+sdist/
+var/
+wheels/
+pip-wheel-metadata/
+share/python-wheels/
+*.egg-info/
+.installed.cfg
+*.egg
+MANIFEST
+
+# Translations	# Translations
+*.pot
+
+# Environments
+.env
+.venv
+env/
+venv/
+ENV/
+env.bak/
+venv.bak/
+env.d/development/crowdin
+
+# Logs
+*.log
+
+# Test & lint
+.coverage
+.pylint.d
+.pytest_cache
+db.sqlite3
+.mypy_cache
+
+# Site media
+/data/
+src/backend/joanie/static/joanie/
+
+# VS Code
+.vscode
+
+# Mac Os
+.DS_Store
```

Comment:
This should be to your `~/.gitignore` file :man_shrugging:

Reply:
I just copy/pasta richie .gitignore but ok I can remove all OS/IDE related items.